### PR TITLE
Story2 mvp

### PR DESCRIPTION
**Pull request by Henry-Tottle**
Ready for trainer review

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ function dataDisplay($dataInput)
     <nav>
         <div>
             <form>
-                <input type="text" placeholder="search books"><input type="submit" value="search">
+                <input type="search" placeholder="search books">
```
Comment (matt-ioa) [inline index.php:46]: Little bit of scope creep here, as this isn't part of story 2! You can leave it in, though.

File: log_book.php [hunk 2]
```diff
@@ -66,32 +68,60 @@
...
-    $author = explode(' ', $author);
-    $isbn = trim($_POST['isbn']);
+if (isset($_POST['title']) && isset($_POST['forename'])
+    && isset($_POST['surname']) && isset($_POST['isbn'])
+    && isset($_POST['genre_1']) && isset($_POST['image']))
+{
+
+
+        $title = trim($_POST['title']);
+
```
Comment (matt-ioa) [inline log_book.php]: Tidy up these blank lines & weird indentation

File: log_book.php [hunk 3]
```diff
@@ -66,32 +68,60 @@
...
 
 
-if (isset($_POST['title'])) {
-    $title = trim($_POST['title']);
-    $author = trim($_POST['author']);
-    $author = explode(' ', $author);
-    $isbn = trim($_POST['isbn']);
+if (isset($_POST['title']) && isset($_POST['forename'])
+    && isset($_POST['surname']) && isset($_POST['isbn'])
+    && isset($_POST['genre_1']) && isset($_POST['image']))
```
Comment (matt-ioa) [inline log_book.php]: You can actually just do this:

```
if (isset($_POST['title'], $_POST['forename'], $_POST['surname'], $_POST['isbn'], $_POST['genre_1'], $_POST['image']))
```

File: log_book.php [hunk 4]
```diff
@@ -66,32 +68,60 @@
...
+    if (ctype_digit($_POST['isbn']))
+    {
+        $isbn = trim($_POST['isbn']);
+    }
     $format = $_POST['format'];
-    $publisher = trim($_POST['publisher']);
+
+
+        $publisher = trim($_POST['publisher']);
+
```
Comment (matt-ioa) [inline log_book.php]: More blank lines to remove & indentation is strange again

File: log_book.php [hunk 5]
```diff
@@ -66,32 +68,60 @@
...
+    }
+    if (ctype_alpha($_POST['genre_3']))
+    {
+        $genre3 = trim($_POST['genre_3']);
+    }
+
     $image = $_POST['image'];
     $rating = $_POST['rating'];
 
     $query = $db->prepare('INSERT INTO `authors` (`forename`, `surname`) VALUES (:forname,:surname)');
```
Comment (matt-ioa) [inline log_book.php]: Looks like a slight typo here: you want `:forename` instead of `:forname`

File: styles.css [hunk 6]
```diff
@@ -30,13 +30,13 @@ body {
 nav {
     display: flex;
     justify-content: space-evenly;
-    background-color: chocolate;
-    padding: 0.5rem 0;
+
+    background: rgb(231,180,11);
```
Comment (matt-ioa) [inline styles.css]: I think this first background property is redundant, as the second one will just override it

File: styles.css [hunk 7]
```diff
@@ -54,8 +54,12 @@ button {
     flex-direction: column;
     align-items: center;
     padding-bottom: 2rem;
-    background-color: cornflowerblue;
-}
+    color: #ffffff;
+    text-shadow: 5px 2px 8px rgba(19,15,2,0.6);
+
+    font-size: 30pt;
+    background: rgb(0,168,255);
```
Comment (matt-ioa) [inline styles.css]: Same here

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Great job, just a couple of minor issues to fix before approval 👍

File: log_book.php (hunk 3 above)
Comment (Henry-Tottle) [reply log_book.php]: Awesome
